test/e2e/atlas/helper_test.go

```diff
@@ -394,6 +414,122 @@ func createProject(projectName string) (string, error) {
 	return project.ID, nil
 }
 
+func createProjectWithoutAlertSettings(projectName string) (string, error) {
+	cliPath, err := e2e.AtlasCLIBin()
+	if err != nil {
+		return "", fmt.Errorf("%w: invalid bin", err)
+	}
+	args := []string{
+		projectEntity,
+		"create",
+		projectName,
+		"-o=json",
+		"--withoutDefaultAlertSettings",
+	}
+	if Gov() {
+		args = append(args, "--govCloudRegionsOnly")
+	}
+	cmd := exec.Command(cliPath, args...)
+	cmd.Env = os.Environ()
+	resp, err := cmd.CombinedOutput()
+	if err != nil {
+		return "", fmt.Errorf("%s (%w)", string(resp), err)
+	}
+
+	var project mongodbatlas.Project
```

Comment:
golint may be in a dirty state, may be a good idea to merge master if you haven't